Assets/CritterController.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
+
+public class CritterController : MonoBehaviour
+{
+    public float consumed, hydrated;
+    public float consumptionLostPerSecond;
+    public float hydrationLostPerSecond;
+    public float force;
+
+    public GameObject target;
+    private Rigidbody rb;
+
+    void Start()
+    {
+        gameObject.GetComponent<Renderer>().material.color = Color.red;
+
+        rb = gameObject.GetComponent<Rigidbody>();
+    }
+
+    void Update()
+    {
+        consumed -= consumptionLostPerSecond * Time.deltaTime;
+        hydrated -= hydrationLostPerSecond * Time.deltaTime;
```

Comment:
Maybe this weekend a good way to open the meeting would be to explain Time.deltaTime - I think it's a key representative of the new ballgame we're playing by porting to Unity, and for people who haven't used a game engine before it may help them get their head in the game.